WordPress/gutenberg
# Change "Swap" to "Replace" for Template Action

## What?
Fixes https://github.com/WordPress/gutenberg/issues/56845
Updates the UI text from "Swap template" to "Replace template" for consistency across Gutenberg and to align with similar actions in the interface.

## Why?
The word "replace" is commonly used in action items throughout the Gutenberg editor (e.g., replacing template parts). Using "swap" creates an inconsistency in terminology and could cause confusion.

Aligning this action with the existing terminology improves clarity and enhances the user experience by providing a consistent language framework.

## How?
Replaced occurrences of "Swap template" with "Replace template" in the UI.

## Testing Instructions

1. Go to the Site Editor.
2. Select Pages and click on a page to edit.
3. Open the Templates dropdown on the right-hand side of the screen.
4. Verify that the action labeled "Swap template" is now labeled "Replace template".
5. Click on the "Replace template" option and confirm that it functions as expected, with no changes to behaviour.

### Testing Instructions for Keyboard


## Screenshots or screencast 

## Before

![Swap_template](https://github.com/user-attachments/assets/351f99a4-c27c-42ac-9b30-94e6b19d621f)

## After

![replace_template](https://github.com/user-attachments/assets/37f95c55-6f32-4e41-94bf-b3a7aa77aa91)
